bigquery/indexer.py

```diff
@@ -0,0 +1,138 @@
+"""Loads BigQuery table into Elasticsearch.
+
+Note: Elasticsearch index is deleted before indexing.
+"""
+
+import argparse
+import csv
+import json
+import logging
+import os
+import time
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+import pandas as pd
+
+
+# Log to stderr.
+logging.basicConfig(level=logging.INFO,
+    format='%(asctime)s %(filename)10s:%(lineno)s %(levelname)s %(message)s',
+    datefmt='%Y%m%d%H:%M:%S')
+logger = logging.getLogger('indexer.bigquery')
+
+
+def parse_args():
+  parser = argparse.ArgumentParser()
+  parser.add_argument(
+      '--elasticsearch_url',
+      type=str,
+      help='Elasticsearch url. Must start with http://',
+      default='http://localhost:9200')
+  parser.add_argument(
```

Comment:
I'm probably missing context here, why not pass in the config file itself? Do we always require the config file to be named 'dataset.json' and/or do we have requirements around what else can/should exist in that directory?

Reply:
Yes, we expect `dataset.json`. I added more details to README. Instructions for what goes in the files are in the files themselves.